Sources/Core/Client/Client+Request.swift

```diff
@@ -56,6 +56,38 @@ extension Client {
         }
     }
     
+    
+    func checkLatestVersion() {
+         #if DEBUG
+        // Check latest pod version and log warning if there's a new version
+        guard let podUrl = URL(string: "https://trunk.cocoapods.org/api/v1/pods/StreamChat") else { return }
+        
+        // swiftlint:disable nesting
+        struct PodTrunk: Codable {
+            struct Version: Codable {
+                let name: String
+            }
+            
+            let versions: [Version]
+        }
+        // swiftlint:enable nesting
+        
+        let versionTask = URLSession(configuration: .default).dataTask(with: podUrl) { data, _, error in
+            guard let data = data, error == nil else {
+                return
+            }
+            do {
+                let podTrunk = try JSONDecoder().decode(PodTrunk.self, from: data)
+                if let latestVersion = podTrunk.versions.last?.name, latestVersion > Client.version {
+                    ClientLogger.logger("📢", "", "StreamChat \(latestVersion) is released (you are on \(Client.version))."
```

Comment:
We decide to print it instead of logging.

Reply:
By default, `ClientLogger.logger` will `print` it anyway, and by using this we're making sure if they're able to customize this.